libs/sdk-ui-charts/src/charts/headline/internal/headlines/baseHeadline/BaseHeadline.tsx

```diff
@@ -41,11 +44,30 @@ const BaseHeadline: React.FC<IHeadlineProps> = ({ data, config, onDrill, onAfter
     );
 
     useEffect(() => {
-        onAfterRender();
-    });
+        // guard if onResize would fail to resize the widget
+        const timeoutId = setTimeout(() => {
+            if (!afterRenderCalled.current) {
+                onAfterRender?.();
+                afterRenderCalled.current = true;
+            }
+        }, RESIZE_GUARD_TIMEOUT);
+
+        return () => clearTimeout(timeoutId);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why we disable hook dependency check ?

Reply:
I wanted to make sure that this scheduled effect runs really only once.

Reply:
Ok byt iw will failed if someone will change onAfterRender its ok ?

Reply:
added onAfterRender to the dependencies, realized that it could be ok to prolong the interval in case the method changes so that we make sure it's really called with the latest handler.